database/schema/controller.go

```diff
@@ -56,6 +68,48 @@ ON lease_pin (lease_uuid, entity_id);
 CREATE INDEX idx_lease_pin_lease
 ON lease_pin (lease_uuid);
 `[1:]
+}
+
+func changeLogSchema() string {
+	return `
+CREATE TABLE change_log_edit_type (
+    id        INT PRIMARY KEY,
+    edit_type TEXT
+);
+
+CREATE UNIQUE INDEX idx_change_log_edit_type_edit_type
+ON change_log_edit_type (edit_type);
+
+-- The change log type values are bitmasks, so that multiple types can be
+-- expressed when looking for changes.
+INSERT INTO change_log_edit_type VALUES
+    (1, 'create'),
+    (2, 'update'),
+    (4, 'delete');
+
+CREATE TABLE change_log_namespace (
+    id        INT PRIMARY KEY,
+    namespace TEXT
+);
+
+CREATE UNIQUE INDEX idx_change_log_namespace_namespace
+ON change_log_namespace (namespace);
+
+-- TODO (stickupkid): Add the namespaces we want to track.
+-- INSERT INTO change_log_namespace VALUES
+--    (1, 'foo');
 
-	return strings.Split(delta, ";\n\n")
+CREATE TABLE change_log (
```

Comment:
After we contrive the base watcher, we need an exercise in creating the correct indexes based on the access patterns.